### PR TITLE
Auto deploy pr 

### PR DESCRIPTION
**Pull request by YashKumarVerma**
@niccokunzmann 
*** merge if extremely urgent***
I have to squash commits. Rest is almost done. 

- [ ] Included a preview link and screenshot showing after and before the changes.
- [ ] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [ ] Included a description of change below.
- [x] Tried to [squash the commits][squash].

### Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link**
  
  http://rawgit.com/USERNAME/labyrinth/master/index.html

- Fixes #<Add issue number here>

### Screenshots

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

**Comment by niccokunzmann** [comment]
The build is failing.

I think, we do not need to commit the `node_modules` but rather use an `npm install` in travis.
What do you think?

 `node_modules` can be added to the .gitignore.

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export SURGE_TOKEN=d1c28a7a75967cc2b4c852cca0d12206
+
+#PUSH
+if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+    surge --project ./ --domain https://fossasia-labyrinth.surge.sh
+    exit 0
+fi
+
+# PR
+export DEPLOY_DOMAIN=https://pr-${TRAVIS_PULL_REQUEST}-fossasia-labyrinth.surge.sh
```
Comment (niccokunzmann) [inline deploy.sh]: awesome!

**Comment by niccokunzmann** [comment]
Ignore my last comment. We are on the same page here. I wonder why it fails...

**Comment by niccokunzmann** [comment]
I think, adding a 

`|node_modules` would do here to remove the failures:
https://github.com/fossasia/labyrinth/blob/master/scripts/check-html-syntax-errors.sh#L14

**Comment by niccokunzmann** [comment]
I tried to make a pull request to your branch but I landed on a 404 page.

**Comment by YashKumarVerma** [comment]
> The build is failing.
> 
> I think, we do not need to commit the `node_modules` but rather use an `npm install` in travis.
> What do you think?
> 
> `node_modules` can be added to the .gitignore.

I think i have not included the node_modules folder. there are only 5 files changed

**Pull request closed without merging**